README.md

```diff
@@ -108,24 +119,137 @@ The compiled workflow can be executed via the DNAnexus command line client or we
 
 ![this](doc/bam_chrom_counter.png)
 
-### Strict syntax
+### Validate the workflow
 
 dxCompiler uses [wdlTools](https://github.com/dnanexus/wdlTools), a parser that adheres strictly to the WDL specifications. Most of the problematic automatic type conversions that are allowed by some other WDL runtime engines are not allowed by dxCompiler. Please use the command line tools in wdlTools (e.g. `check` and `lint`) to validate your WDL files before trying to compile them with dxCompiler.
 
 ## CWL
 
-To compile a CWL workflow/tool, you will first need to process it using the following steps:
+### Preprocess CWL workflow
```

Comment:
I'd like to provide real examples of the 3 steps listed in this section.

Reply:
Maybe I can put the example cwl files (v1.0, v1.2 and packed json) in contrib/beginner_example and add the result link under each step?

Reply:
I was thinking of actually changing the example to illustrate all 3 steps.